src/main/java/in/twizmwaz/cardinal/util/FireworkUtil.java

```diff
@@ -0,0 +1,58 @@
+package in.twizmwaz.cardinal.util;
+
+import org.bukkit.Color;
+import org.bukkit.FireworkEffect;
+import org.bukkit.Location;
+import org.bukkit.World;
+import org.bukkit.entity.Firework;
+import org.bukkit.inventory.meta.FireworkMeta;
+
+import java.util.Random;
+
+public class FireworkUtil {
+
+    public static void spawnFirework(Location location, World world) {
+        Firework firework = (Firework) world.spawn(location, Firework.class);
+        FireworkMeta fireworkMeta = firework.getFireworkMeta();
+        Random random = new Random();
+        int randomColor = random.nextInt(17);
+        FireworkEffect effect = FireworkEffect.builder().withColor(randomColor(randomColor)).with(FireworkEffect.Type.BALL).trail(true).build();
+        fireworkMeta.addEffect(effect);
+        fireworkMeta.setPower(0);
+        firework.setFireworkMeta(fireworkMeta);
+    }
+
+    public static void spawnFirework(Location location, World world, Color color) {
+        Firework firework = (Firework) world.spawn(location, Firework.class);
+        FireworkMeta fireworkMeta = firework.getFireworkMeta();
+        FireworkEffect effect = FireworkEffect.builder().withColor(color).with(FireworkEffect.Type.BALL).trail(true).build();
+        fireworkMeta.addEffect(effect);
+        fireworkMeta.setPower(0);
+        firework.setFireworkMeta(fireworkMeta);
+    }
+
+    private static Color randomColor(int c) {
```

Comment:
Why the int if it is random? Random.nextInt(16) will return a pseudo-random int between 0-16.